Classroom account (#5174)
Qiskit/documentation: docs/guides/classroom-accounts.mdx

</Admonition>

## Activate the account

Approved Classroom Account applicants will receive an email that contains a 32-digit feature code. Do not delete this email; it contains important information for submitting your new account ID once you've set up your account, which is a required step to complete the activation. 

Once you have a feature code, follow the activation steps below:

<Admonition type="caution" title="Do not enter credit card details">
When creating an account with a feature code, do not provide credit card information on IBM Cloud&reg; unless you are actively upgrading to a paid account and are willing to accept being billed for consumed services.
</Admonition>

1. Visit https://cloud.ibm.com/.
1. Click **Create account**. You must create a new account even if you have a personal IBM Cloud account, because the Classroom Account is not attached to your personal account.

> 1. Enter a valid email address and a password. Use a different email address from one associated with an existing IBM Cloud account, if you already have one. Click **Next**.

Reviewer: ```suggestion
1. Enter a valid email address and a password. Use an email address that is **not** associated with an existing IBM Cloud account. Click **Next**.
```

Author: Weird, I thought I made this change!!!

1. Check your email inbox for a seven-digit verification code, then return to the account creation form. Enter that code to verify your new account.

> 1. **Important:** In the Personal information section, do not enter your first and last name. Instead, in the "First name" field, enter your institutional affiliation, and in the "Last name" field, enter the name of the course using the account. Enter your country or region.

Reviewer: Would they need separate accounts for multiple courses then?

Author: I think so, yes? We should add detail for that in a followup - this needs to go out

1. Complete the Account notice and click **Continue**.
1. Complete the Account privacy notice and click **Continue**.

> 1. At the Verify identity screen, click on the **Register with a code** option. 

Reviewer: ```suggestion
1. At the Verify identity screen, click **Register with a code**. 
```

> 1. Enter your 32-digit feature code in the **Enter code** text box. Click **Create account** to proceed.

Reviewer: ```suggestion
1. Enter your 32-digit feature code from your confirmation email in the **Enter code** text box, then click **Create account**.
```

1. Navigate to your [account settings on IBM Quantum Platform](https://quantum.cloud.ibm.com/account) and copy the Account ID (found under Details). 

> 1. Follow the instructions in the feature code email you received to send this ID number to the IBM Quantum team, and they will process the Classroom Account for full activation.

Reviewer: ```suggestion
1. Follow the instructions in the confirmation email you received to send this ID number to the IBM Quantum team, and they will process the Classroom Account for full activation.
```

## Add users

> 1. First, ensure you have the correct account and region selected in the account and region switchers in the header.

Reviewer: ```suggestion
1. First, ensure you have the correct account and region selected in the account and region switchers in the[IBM Quantum Platform](https://quantum.cloud.ibm.com) header.
```

1. Select [**Access management**](https://quantum.cloud.ibm.com/users) from the main menu.
1. On the Users tab, click "Invite user".

> 1. Enter the users' email addresses. You can add multiple addresses at once.

Reviewer: ```suggestion
1. Enter the users' email addresses. You can add multiple addresses at the same time.
```

Reviewer: separated by commas?

Author: The field they see specifies that they can do it multiple ways.

> 1. By default, an instance is assigned to each new user. It is recommended to use the default, rather than turning off that option with the toggle. 

Reviewer: ```suggestion
1. By default, an instance is assigned to each new user. It is recommended to accept the default setting.
```

1. After the user accepts the invitation and creates their account, they can navigate to [IBM Quantum Platform](https://quantum.cloud.ibm.com/), where they will see their Classroom Account instance in the account switcher in the page header.

## Remove users

To remove a user from a Classroom Account, follow these steps:

1. Select [**Access management**](https://quantum.cloud.ibm.com/users) from the main menu.
1. On the Users tab, click the three-dot menu at the end of that user's row. 
1. Select Remove user. You will see a prompt to confirm that you want to remove the user from the account. Click Remove to confirm.

## Manage instances

Each instance provides up to 10 minutes of QPU usage per 28-day rolling window.

As the Classroom Account owner, you can view all activity across all instances in the account on the [Analytics](https://quantum.cloud.ibm.com/analytics) page.

Student users can see data from workloads run in their own instance. They cannot see data from any other instances.

## Renew the Classroom Account

> To renew a Classroom Account that is nearing its expiration date, request a new feature code using this [renewal form](https://ibm.biz/account-renewal). 

Reviewer: How long does it last?  Do they get a warning?

## Next steps

<Admonition type="tip" title="Recommendations">

> Direct your students to the [documentation](/docs/guides/compute-services) for details on how to use IBM Quantum Platform. Topics include the following:

Reviewer: ```suggestion
Direct your students to the [documentation](/docs/guides/compute-services) for details on how to use IBM Quantum Platform. Commonly used topics include the following:
```
